lib/core/test/unit/Cardano/Wallet/Primitive/AddressDerivationSpec.hs

```diff
@@ -340,6 +341,17 @@ instance {-# OVERLAPS #-} Arbitrary (Passphrase "encryption") where
         bytes <- T.encodeUtf8 . T.pack <$> replicateM n arbitraryPrintableChar
         return $ Passphrase $ BA.convert bytes
 
+instance {-# OVERLAPS #-} Arbitrary (Passphrase "seed") where
+    arbitrary = do
+        n <- choose (5, 255)
+        bytes <-
+            T.encodeUtf8 . T.pack <$> replicateM n arbitraryPrintableChar
+            `suchThat`
+            (\bs ->
+                 let bsLength = (BS.length . T.encodeUtf8 . T.pack) bs
+                 in bsLength >= 16 &&  bsLength <= 255)
+        return $ Passphrase $ BA.convert bytes
```

Comment:
I think this instance has no use anymore, doesn't it :thinking: ?

Reply:
Fixed in: https://github.com/input-output-hk/cardano-wallet/pull/381/commits/a3cba7d819b31b72cd33cdf5c6d285b95f7236c3

Reply:
I believe this instance is needed. General one is too broad and we will get password generated with lenght smaller than 16 or higher than 255 easily

Reply:
in the absensce of this one the following is responsible for generation : 
```
328 instance Arbitrary (Passphrase goal) where                                                                                                                                                
329     shrink (Passphrase "") = []                                                                                                                                                           
330     shrink (Passphrase _ ) = [Passphrase ""]                                                                                                                                              
331     arbitrary = do                                                                                                                                                                        
332         n <- choose (0, 32)                                                                                                                                                               
333         InfiniteList bytes _ <- arbitrary                                                                                                                                                 
334         return $ Passphrase $ BA.convert $ BS.pack $ take n bytes
```